ui/component/settingWalletServer/view.jsx

```diff
@@ -130,6 +131,16 @@ function SettingWalletServer(props: Props) {
           }}
           label={__('Use custom wallet servers')}
         />
+        <p className="help">
+          <I18nMessage
+            tokens={{
+              help_link: <Button button="link" href="http://lbry.com/faq/wallet-servers" label={__('Learn More')} />,
+            }}
+          >
+            Wallet servers control what content is trending, ..., idk. %help_link%.
```

Comment:
Looking for copy here @tzarebczan @kauffj

Reply:
Wallet servers are used to relay data to and from the LBRY blockchain. They may also configure different trending algorithms, content filters, and blocklists.